src/main/java/de/zalando/aruha/nakadi/controller/VersionController.java

```diff
@@ -0,0 +1,98 @@
+package de.zalando.aruha.nakadi.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import javax.ws.rs.core.MediaType;
+import org.codehaus.jackson.annotate.JsonProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestMapping;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/version", produces = MediaType.APPLICATION_JSON)
+public class VersionController {
+
+    private static final String SCM_SOURCE_FILE = "/scm-source.json";
+
+    private static class ScmSource {
+        @JsonProperty("author")
+        private String author;
+        @JsonProperty("revision")
+        private String revision;
+        @JsonProperty("status")
+        private String status;
+        @JsonProperty("url")
+        private String url;
+
+        public String getAuthor() {
+            return author;
+        }
+
+        public void setAuthor(String author) {
```

Comment:
Yes, I think in all Java teams here it's an unofficial rule that everything that can be immutable/final should be immutable/final.